content/docs/en/getting-started/1-quick-start.md

```diff
@@ -9,19 +9,22 @@ it in the [NativeScript Playground](https://play.nativescript.org?template=play-
 If you are ready to move on and work locally on your machine, and [you
 have your system ready for native development](/en/docs/getting-started/installation), you can start by using a template:
 
-```shell
-$ tns create my-app --template nativescript-vue-template
-$ cd my-app
+###  Scaffold project
+```sh
+$ npm install -g vue-cli
+$ vue init nativescript-vue/vue-cli-template <project-name>
+$ cd <project-name>
 ```
 
-and then run your app using:
-
-```shell
-$ tns run android # for android
+### Install dependencies
+```sh
+$ npm install
 ```
 
-```shell
-$ tns run ios # for ios
+### Build, watch for changes and run the application
+Platform below would be `ios` or `android`
+```sh
+$ npm run watch:<platform>
```

Comment:
Might be a good idea to mention the `build` command as well, and then link to the template for more details.

Reply:
I purposely left out the build command because its instructions to install / get started what do you think?. But I think the link should be there I'll add it